### DISCUSSION TITLE
Using Data tables inside content tabs

### DISCUSSION THREAD

**Discussion by bitfoo1** [squidfunk/mkdocs-material#5767] [Q&A] [answered]
Hi, 

I am using the content tabs. But using [data tables](https://squidfunk.github.io/mkdocs-material/reference/data-tables/) inside content tabs will not render the table. It renders the actual text with all markdown meta characters.

Example:

```markdown
=== "foo"
    Fließtext oder tabellarisch
    | Header 1 | Header 2 | Header3  |
    | :------: | :------- | :------- |
    |    1     | Sample 1 | Sample 3 |
    |    2     | Sample 2 | Sample 4 |
```
is rendering to 
<img width="1190" alt="Screenshot 2023-07-27 at 13 13 09" src="https://github.com/squidfunk/mkdocs-material/assets/3464646/9c3558b4-e767-4d15-917f-61625a07ad4f">

**Comment by facelessuser** [comment, marked as answer]
Try putting a new line between `Fließtext oder tabellarisch` and the start of the table.

**Reply by bitfoo1** [reply]
Thank you! That does the trick.
